altos-rust/altos-core/src/task/control.rs

```diff
@@ -53,8 +53,14 @@ pub enum Priority {
   /// rely on any operation to be atomic unless specifically marked in a critical section.
   Normal = 1,
 
+  /// The minimal task priority.
+  ///   
+  /// These tasks should be purely optional for the system to run. They will only be run if there
+  /// are no other tasks to run, so on some systems they may never run at all.
```

Comment:
This isn't true any more now right? They're guaranteed to run at least after a little while